custom_components/tuya_local/devices/electriq_sc16hpw_aircon.yaml

```diff
@@ -0,0 +1,216 @@
+name: ElectriQ SupremeCool SC16HPW Air Conditioner (XKA-G02)
+#multi_dps_update: false
+entities:
+  - entity: climate
+    dps:
+      - id: 1
+        name: hvac_mode
+        type: boolean
+        mapping:
+          - dps_val: true
+            value_mirror: current_hvac_mode
+          - dps_val: false
+            value: 'off'
+      - id: 2
+        name: current_temperature
+        type: integer
+        mapping:
+          - scale: 1
```

Comment:
A scale of 1 is redundant, it is only needed inside a conditional block to override a different scale in special cases